po/POTFILES.in

```diff
@@ -8,5 +8,6 @@ plugins/uefi/fu-plugin-uefi.c
 src/fu-config.c
 src/fu-debug.c
 src/fu-main.c
+src/fu-noinst.c
```

Comment:
This one snuck in and shouldn't be part of this PR.

Reply:
Oops, thanks. Fixed.